ocis/tests/acceptance/features/apiOcisSpecific/apiVersions-fileVersions.feature

```diff
@@ -23,7 +23,7 @@ Feature: dav-versions
     And user "Alice" uploads file "filesForUpload/davtest.txt" to filenames based on "/davtest.txt" with all mechanisms using the WebDAV API
     Then the version folder of file "/davtest.txt-olddav-regular" for user "Alice" should contain "1" element
     And the version folder of file "/davtest.txt-newdav-regular" for user "Alice" should contain "1" element
-    And as "Alice" file "/davtest.txt-olddav-oldchunking" should not exist
+    Then the version folder of file "/davtest.txt-olddav-oldchunking" for user "Alice" should contain "0" element
```

Comment:
https://drone.owncloud.com/owncloud/ocis/1451/14/7
It returned 1 element in CI
```suggestion
    Then the version folder of file "/davtest.txt-olddav-oldchunking" for user "Alice" should contain "1" element
```

Reply:
yep, but it flips with the other storage. I think I have that solved in my next commit.

I'm more puzzled by:
```

ℹ Connected to selenium on port 4444 (500ms).
--
992 | tests/acceptance/features/webUISharingInternalUsers/shareWithUsers.feature:486
993 |  
994 | Given the setting "shareapi_auto_accept_share" of app "core" has been set to "no"
995 | And the administrator has set the default folder for received shares to "Shares"
996 | And these users have been created with default attributes:
997 | │ username │
998 | │ user1    │
999 | │ user2    │
1000 | Given user "user1" has shared folder "simple-folder" with user "user2"
1001 | And user "user2" has accepted the share "simple-folder" offered by user "user1"
1002 | When user "user2" has logged in using the webUI
1003 | √ Element <input[autocomplete="kopano-account username"]> was visible after 597 milliseconds.
1004 | √ Element <#oc-app-container> was visible after 2605 milliseconds.
1005 | √ Element <//button[@id="new-file-menu-btn" and contains(@class, "oc-button-primary") and not(@disabled)]> was present after 1932 milliseconds.
1006 | √ Element <//div[contains(@class, "oc-file") and @data-preview-loaded="false"]> was not present after 1506 milliseconds.
1007 | And the user opens folder "Shares" using the webUI
1008 | √ Element <#files-list-container> was present after 26 milliseconds.
1009 | √ Element <.vue-recycle-scroller> was visible after 46 milliseconds.
1010 | √ Element <.vue-recycle-scroller> was visible after 28 milliseconds.
1011 | √ Element <//span[contains(@class, "oc-file-name") and text()='Shares' and not(../span[contains(@class, "oc-file-extension")])]/ancestor::div[@data-is-visible="true"]//div[contains(@class, "oc-file") and (@data-preview-loaded="true" or @data-preview-loaded="disabled")]> was visible after 46 milliseconds.
1012 | √ Element <#files-list-progress> was not present after 3124 milliseconds.
1013 | √ Element <//div[contains(@class, "oc-file") and @data-preview-loaded="false"]> was not present after 19 milliseconds.
1014 | And the user opens the sharing sidebar for folder "simple-folder"
1015 | √ Element <#files-list-container> was present after 15 milliseconds.
1016 | √ Element <.vue-recycle-scroller> was visible after 41 milliseconds.
1017 | √ Element <.vue-recycle-scroller> was visible after 36 milliseconds.
1018 | √ Element <//span[contains(@class, "oc-file-name") and text()='simple-folder' and not(../span[contains(@class, "oc-file-extension")])]/ancestor::div[@data-is-visible="true"]//div[contains(@class, "oc-file") and (@data-preview-loaded="true" or @data-preview-loaded="disabled")]> was visible after 49 milliseconds.
1019 | (node:391) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 queue:finished listeners added. Use emitter.setMaxListeners() to increase limit
1020 | Then the following resources should have share indicators on the webUI
1021 | │ fileName      │ expectedIndicators │
1022 | │ simple-folder │ user-indirect      │
1023 | √ Element <#files-list-container> was present after 17 milliseconds.
1024 | √ Element <.vue-recycle-scroller> was visible after 38 milliseconds.
1025 | √ Element <.vue-recycle-scroller> was visible after 36 milliseconds.
1026 | √ Element <//span[contains(@class, "oc-file-name") and text()='simple-folder' and not(../span[contains(@class, "oc-file-extension")])]/ancestor::div[@data-is-visible="true"]//div[contains(@class, "oc-file") and (@data-preview-loaded="true" or @data-preview-loaded="disabled")]> was visible after 46 milliseconds.
1027 | ✖ failed
1028 | AssertionError [ERR_ASSERTION]: Expected share indicators to be the same for "simple-folder": expected [user-indirect] got []
1029 | + expected - actual
1030 |  
1031 | -false
1032 | +true
1033 |  
1034 | at _callee58$ (/srv/app/phoenix/tests/acceptance/stepDefinitions/filesContext.js:1081:12)
1035 | at tryCatch (/srv/app/phoenix/node_modules/regenerator-runtime/runtime.js:63:40)
1036 | at Generator.invoke [as _invoke] (/srv/app/phoenix/node_modules/regenerator-runtime/runtime.js:293:22)
1037 | at Generator.next (/srv/app/phoenix/node_modules/regenerator-runtime/runtime.js:118:21)
1038 | at asyncGeneratorStep (/srv/app/phoenix/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)
1039 | at _next (/srv/app/phoenix/node_modules/@babel/runtime/helpers/asyncToGenerator.js:25:9)
1040 | at processTicksAndRejections (internal/process/task_queues.js:86:5)
1041 | When the user opens folder "simple-folder" using the webUI
1042 | - skipped
1043 | Then the following resources should have share indicators on the webUI
1044 | │ fileName            │ expectedIndicators │
1045 | │ simple-empty-folder │ user-indirect      │
1046 | │ lorem.txt           │ user-indirect      │
1047 | - skipped
1048 | deleted Share
```

Reply:
@kulmann @LukasHirt what about the share indicators ... I know they don't work on ocis, yet ... I can try to fix that, but I'd rather get this PR in ... now ...

Reply:
Currently broken in phoenix master: https://github.com/owncloud/phoenix/issues/4310
Skip it if possible.

Reply:
Currently the Phoenix UI tests are only run with `owncloud` storage. So I don't understand? How did it break in CI?

Reply:
ok, share indicators in the file list rely on the `<oc:share-types>` property, which is not implemented in ocis, yet.

Reply:
@kulmann wasn't the Phoenix pr supposed to allow this? Ugh I need to really click through this...